app/src/main/java/com/mobilecoronatracker/ui/about/AboutFragment.kt

```diff
@@ -25,16 +24,15 @@ class AboutFragment : Fragment() {
         val binding = DataBindingUtil.inflate<FragmentAboutBinding>(inflater,
             R.layout.fragment_about, container, false
         )
-        viewModel = CreditsListViewModel(requireActivity().application)
+        viewModel = AboutViewModel(requireActivity().application)
```

Comment:
Please use ViewModelProvider or koin by viewModel to let ViewModel onCleared() be called

Reply:
done.